src/draw/drawings.js

```diff
@@ -71,18 +74,18 @@ function createNonTraversableCellSprites(
  * the corresponding cells in green on the map.
  */
 export function drawPlannedPath(path, cpt, hexColor = 0x00ff00, alpha = 0.25) {
-  clearSprites(tagpro.renderer.pathSprites); // clear the previous path from the map
+  pathSprites = clearSprites(pathSprites); // clear the previous path from the map
```

Comment:
I don't like this setting and re-setting of these global vars.  It's too easy to lose track of what the current sprites is and what it means. Is there another way to do this, can we have the create functions return the sprites that it creates?  And then just have those variables be in the scope of the `draw`* functions?